yb-voyager/cmd/exportData.go

```diff
@@ -303,7 +303,7 @@ func exportData() bool {
 	tableListTuplesToDisplay := lo.Map(finalTableList, func(table sqlname.NameTuple, _ int) sqlname.NameTuple {
 		renamedTable, isRenamed := renameTableIfRequired(table.ForOutput())
 		if isRenamed {
-			t := table.ForMinOutput()
+			t := table.ForOutput()
```

Comment:
what is the reason for changing it from `ForMinOutput` to `ForOutput`

Reply:
this is regarding the 4th UI issue in the above ticket, where for the targe-db-schema non-public also we were displaying the only tables in MinOutput and qualified at some places with Output so made it consistent ForOutput in all scenarios